searchEngine/testing/searchTest.py

```diff
@@ -0,0 +1,69 @@
+import os
+import sys
+import inspect
+
+current_dir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+parent_dir = os.path.dirname(current_dir)
+sys.path.insert(0, parent_dir) 
```

Comment:
I've never seen this done before, seems pretty hacky even for a prototype. Is this common practice for testing in Python?

Reply:
There seems to be a lot of ways to import a parent directory, [this](https://docs.pytest.org/en/stable/goodpractices.html) looks like a good resource for a bit cleaner code.

Reply:
Yeah that was my bad I got really confused between python 2 and 3 due to the __init__.py so I just used the first thing that worked